### ISSUE TITLE
automatically cd  or open folder on goto <path-to-folder>

### ISSUE THREAD

**Issue by technocake** [technocake/goto#1]
there should be a setting that determines gotos behavior on magic words that are a path to folder.
Some users might prefer to open the folder in Explorer | Finder etc.  
You can do this today:

cd in terminal:
 `goto cd <magicword>`

open folder in explorer:
 `goto  -f <magicword>`

goto should understand what the user prefers and call the corresponding handler if we provide the same word (that points to a folder) without the cd or -f argument. 

so TL;DR:
goto word --> automagically opens in folder or in cd.

**Comment by Arxcis** [comment]
I now realize that this issue is closely related to #4.
I think that the default mode should be `cd` and then there could be a `-o`for explorer.
Since `goto` is a command-line tool, `cd` - ing is what i except to be the default behaviour.

My humble API suggestion
```
goto <magicword>            # cd to folder
goto <magicword> -o --open  # open window
```
I am not 100% sure if it is a good idea to do both cd to folder + open window, but in many case it would be useful. Some people might find it annoying. Maybe a third option would be cool
```
goto <magicword>                 # cd
goto <magicword> -o --open       # open
goto <magicword> --ocd           # cd + open  (lol ocd)
```

**Comment by technocake** [comment]
1.  change  flag in the_real_goto.py from -f  to -o, --open and open
2.  update start_goto in _goto_completions to use -o, --open and open instead of -f 

To do 2.  simply append these new flags to the chain of || statements.

**Comment by Arxcis** [comment]
I want to do this as part of a bigger change where I try to use something like the [Click](http://click.pocoo.org/5/) library for command line parsing.

**Comment by technocake** [comment]
For now I implemented it in the brute way the parsing is now,

Will check out Click and see how it is to work with compared to argparse

**Comment by technocake** [comment]
Let's roll with Click 👍
